### DISCUSSION TITLE
Badge Data with getlisting

### DISCUSSION THREAD

**Discussion by muratali016** [etsy/open-api#987] [Ideas]
Hi, I think that's a good idea to add badge data under the getlisting request. I'm creating a system to track my product's badges, however, data that comes from API would be stable.
E.g.
"title":"Macbook Bag.....", "badge":"Bestseller"

**Comment by Gareth064** [comment]
This is something you'd have to email the Etsy Devs about and ask them if they can add it.

Developer@etsy.com

**Comment by muratali016** [comment]
Thanks for the answer :) I'm not doing web scrapping because of that I asked for an API method
